examples/14-community-hosp/README.md

```diff
@@ -0,0 +1,99 @@
+# Community-hospital model
+
+In this model, we have three states:
+
+- Susceptible individuals (in the community),
+- Infected individuals (in the community), and
+- Infected hospitalized individuals.
+
+Susceptible individuals may become hospitalized or not (so they have two possible transitions), and infected individuals may recover or (if hospitalized) stay infected but be discharged (so they go back as infected but in the community).
+
+The model has the following parameters:
+
+- Prob hospitalization: 0.1
+- Prob recovery: 0.33
+- Discharge infected: 0.1
+
+Here is an example of the run
+
+```bash
+root ➜ /workspaces/epiworld/examples/14-community-hosp (example-karim) $ make
+g++ -std=c++17 -O3 -g main.cpp -o main.o
+root ➜ /workspaces/epiworld/examples/14-community-hosp (example-karim) $ ./main.o
```

Comment:
I'd suggest updating this example to remove `(example-karim)` for additional clarity